released/usecases/services_apps/sap_build_code/step1/variables.tf

```
@@ -0,0 +1,114 @@
+variable "globalaccount" {
+  type        = string
+  description = "The globalaccount subdomain where the sub account shall be created."
+}
+
+variable "setup_ai_launchpad" {
+  type        = bool
+  description = "Switch to enable the setup of the AI Launchpad."
+  default     = true
+}
+
+variable "subaccount_name" {
+  type        = string
+  description = "The subaccount name."
+  default     = "My SAP Build Apps subaccount."
+}
+
+variable "cli_server_url" {
+  type        = string
+  description = "The BTP CLI server URL."
+  default     = "https://cpcli.cf.sap.hana.ondemand.com"
```

Here we should use the new endpoint: `https://cli.btp.cloud.sap`

```suggestion
  default     = "https://cli.btp.cloud.sap"
```